Main.sublime-menu

```diff
@@ -13,19 +13,11 @@
                 [
                     {
                         "caption": "PackageResourceViewer",
-                        "children":
-                        [
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/PackageResourceViewer/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – Default"
-                            },
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/User/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – User"
-                            }
-                        ]
+                        "command": "edit_settings",
```

Comment:
Just found something else: we could create a polyfill for it, or just simply propose ST2 user to download the plugin that enables this plugin. It's in here: `Packages/Default/settings.py`.

So, you could in the readme do something like this:

### For sublime text 2 users

1. Copy [this plugin's code](https://raw.githubusercontent.com/twolfson/sublime-files/master/Packages/Default/settings.py)
2. Create a file in your sublime user (`Preferences -> Browse Packages`, and find `User`) directory, and call it `settings.py`
3. Paste the code you've just copied in this file

Matt